File: PRAKTIKUM PEMROGRAMAN I P1/MODUL 5/SOAL 5/PRAK505-2410817210002-Sebastian_Abe_Santoso.c
#include <stdio.h>
void Biodata(int tahunLahir, char Namaku[], char Asal[]){
int tahun_sekarang = 2020;
printf("Perkenalkan Nama Saya : %s\n", Namaku);
printf("Umur saya : %d\n", tahun_sekarang-tahunLahir);
printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
printf("Asal Saya Dari : %s\n", Asal);
}
int main() {
int tahunLahir;
char A[20], B[15];
scanf(" %d",&tahunLahir);
scanf(" %[^\n]%*c",&A);
scanf(" %[^\n]%*c",&B);
Biodata(tahunLahir, A, B);
return 0;
}

File: MODUL 5/SOAL 5/PRAK505-2410817210002-Sebastian_Abe_Santoso.py
def biodata(tahunLahir, Namaku, Asal):
    tahun_sekarang = 2020
    print(f"Perkenalkan Nama Saya : {Namaku}")
    print(f"Umur saya : {tahun_sekarang - tahunLahir}")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya Dari : {Asal}")
tahunLahir = int(input())
namaku = input()
asal = input()
biodata(tahunLahir, namaku, asal)